frontend/src/app/components/containers/pages/not-found.tsx

```diff
@@ -1,3 +1,34 @@
-export const NotFound = () => {
-  return <h1>The page or content you are looking for is not found.</h1>;
+import { FC } from "react";
+import { Footer, Header } from "../../containers/layout";
+import { BsFileEarmarkX } from "react-icons/bs";
+import { Link } from "react-router-dom";
+
+const NotFound: FC = () => {
+  return (
+    <div className="comp-app-container">
+      <Header />
+
+      <div className="error-container">
+        <div className="message">
+          <BsFileEarmarkX />
+          <div className="message-details">
+            <h1 className="comp-padding-top-25">The page or content you are looking for is not found.</h1>
```

Comment:
Nit pick - this is grammatically incorrect. I personally would say "The page or content you are looking for has not been found."

Reply:
updated to "could not be found" as per conversation with Neil.